swap_meet/item.py

```diff
@@ -1,2 +1,31 @@
+import uuid
+
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0):
+        if id:
+            self.id = id
+        else:
+            self.id = int(uuid.uuid4())
+        self.condition = condition
+
+    def get_category(self):
+        class_name = Item()
+        return class_name.__class__.__name__
+
+    def __str__(self):
+        return f"An object of type Item with id {self.id}."
+
+    def condition_description(self):
+        if self.condition == 0:
+            return "cannot reuse"
+        elif self.condition == 1:
+            return "poor"
+        elif self.condition == 2:
+            return "heavily used"
+        elif self.condition == 3:
+            return "good"
+        elif self.condition == 4:
+            return "like-new"
+        elif self.condition == 5:
+            return "new"
```

Comment:
This works fine! It's a lot to read, though. How might we use a list or dictionary to organize our condition string values and have just one return statement?

For example:
```python
conditions = [
    "heavily used",
    "acceptable",
    # so on and so forth
    
    # how do we return the right condition based on the data structure?
```